TF_Application/Assets/TouchFree/Tooling/Scripts/Cursors/DotCursor.cs

```diff
@@ -60,10 +61,13 @@ public class DotCursor : TouchlessCursor
         // transparent. It has to be scaled to match the ring itself.
         public RectTransform ringMask;
 
-        // Variable: ringThickness
-        // Used to set the thickness of the ring itself (i.e. the distance between the inner and
-        // outer edges of the ring)
-        public float ringThickness = 1.5f;
+        // Variable: minRingThickness
+        // The minimum thickness the ring can be.
+        public float minRingThickness = 1.5f;
```

Comment:
I was thinking that, but i'm not enjoying the dependency the base has on rings although because of the thickness value we currently have, there is no getting around it until we have per-cursor configs. Will move to the base for now though

Reply:
Oh I thought they were already _on_ the baseclass and was just commenting that they should be overrides instead of declaring a second member with the same name (making the base class' inaccessible). I think for now, given that all of our cursors use rings, it's fine to have rings on the base class, though we may want to switch that in future. We could even consider making a "RingCursor" _interface_ so we don't have to worry about a middle base-class, if there is no ring-specific but for-every-ring functionality.